### PR TITLE
Add Leipzig

### PR DESCRIPTION
**Pull request by Twissi**
We would like to add Leipzig 🙌

### CODE REVIEW

File: _cities/leipzig.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - norma: Norma
+  - christina: Christina
+  - krissi: Krissi
+---
+
+## About
+
+Norma and Christina started "LearnDjango SelbsthilfeTreff" beginning of 2018 and are now opening up this meetup to all learners and all languages. Together with Krissi we offer a biweekly meetup to learn, collaborate and meet other learners at Basislager in Leipzig. 
+
+We encourage women of all ages to join us.
```
Comment (mamhoff) [inline _cities/leipzig.md]: Can you formulate this sentence slightly more open to people of other genders? Something like
"We encourage learners of all ages to join us, and welcome women and non-binary people in particular."?

**Comment by Twissi** [comment]
@mamhoff updated the copy. Hope to get this merged before the weekend.

**Comment by mamhoff** [comment]
Fantastic! @xMartin Can you merge and deploy (or give me the rights? I have time tomorrow).

**Comment by xMartin** [comment]
Thank you @twissi

**Pull request merged**